libraries/AP_LeakDetector/AP_LeakDetector.cpp

```diff
@@ -0,0 +1,110 @@
+#include "AP_LeakDetector.h"
+
+#include <AP_HAL/AP_HAL.h>
+#include "AP_LeakDetector_Analog.h"
+#include "AP_LeakDetector_Digital.h"
+
+const AP_Param::GroupInfo AP_LeakDetector::var_info[] = {
+
+	// @Param: 1_PIN
+	// @DisplayName: Pin that leak detector is connected to
+	// @Description: Pin that the leak detector is connected to
+    // @Values: -1:Disabled,50:Pixhawk Aux1,51:Pixhawk Aux2,52:Pixhawk Aux3,53:Pixhawk Aux4,54:Pixhawk Aux5,55:Pixhawk Aux6,13:Pixhawk 3.3ADC1,14:Pixhawk 3.3ADC2,15:Pixhawk 6.6ADC
+	// @User: Standard
+    AP_GROUPINFO("1_PIN", 0, AP_LeakDetector, _pin[0], -1),
```

Comment:
first var in table should be 1
note that this would break existing setups (existing eeprom values would break). If that is important for ArduSub we can discuss workarounds